Lesson_01/05Lesson/task01.cs

```diff
@@ -0,0 +1,127 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Lessons._05Lesson
+{
+    /// <summary>
+    /// Создать класс рациональных чисел. В классе два поля – числитель и знаменатель. 
+    /// Предусмотреть конструктор. Определить операторы ==, != (метод Equals()), <, >, 
+    /// <=, >=, +, –, ++, --. Переопределить метод ToString() для вывода дроби. 
+    /// Определить операторы преобразования типов между типом дробь,float, int. 
+    /// Определить операторы *, /, %.
+    /// </summary>
+    public class Digits : IComparable<Digits>
```

Comment:
Подобные вещи (комплексные числа, вектора, дроби) - реализуют обычно в виде структур
```suggestion
    public readonly struct Digits : IComparable<Digits>
```